documentation/source/development/building.rst

```diff
@@ -163,7 +169,7 @@ Install dependencies and tools:
 
 __ https://github.com/inexorgame/vulkan-renderer/blob/master/documentation/source/development/building.rst
 
-.. _Install the SDK: https://vulkan.lunarg.com/doc/sdk/latest/linux/getting_started_ubuntu.html#user-content-install-the-sdk
+.. _Install the SDK: https://vulkan.lunarg.com/doc/sdk/latest/linux/getting_started_ubuntu.html
```

Comment:
We could also replace that with `https://vulkan.lunarg.com/doc/view/latest/linux/getting_started_ubuntu.html#user-content-install-the-sdk`. That would _really_ bring the user to the correct location. :)

Reply:
Doesn't works anymore?